app/templates/views/home.html

```diff
@@ -109,7 +109,7 @@ <h1 class="display-4"> WHAT'S ON </h1>
               <img class="card-img-top cardimgtopmulti" src="{{ images_url }}{{ all_events[count].image_filename }}">
               {% if event.get('past') %}
                 <div class="card-body past_card_padding cream_backgroundcolour">
-                <div class="past_corner"><span>Past Event</span></div>
+                <div class="past_corner"><span id="past_event">Past Event</span></div>
```

Comment:
Is `past_event` going to be used multiple times? It should be a class rather than an id in that case